internal/workflow.go

```diff
@@ -83,36 +99,69 @@ type (
 		//   decoding will be attempted, so you can try it yourself.
 		// - for other channels, an interface{} pointer.  All values are interfaces, so this will never fail, and you
 		//   can inspect the type with reflection or type assertions.
-		Receive(ctx Context, valuePtr interface{}) (more bool)
+		Receive(ctx Context, valuePtr interface{}) (ok bool)
```

Comment:
more consistent ok/more use across the board to match the go language reference's variable names.